### PR TITLE
Remove WinForms/WPF BlazorWebView controls

### PR DESCRIPTION
**Pull request by Eilon**
They are now in the dotnet/maui repo: https://github.com/dotnet/maui/pull/1199

### CODE REVIEW

File: eng/ProjectReferences.props [hunk 1]
```diff
@@ -150,10 +150,7 @@
...
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView.WebView2" ProjectPath="$(RepoRoot)src\Components\WebView\Platforms\WebView2\src\Microsoft.AspNetCore.Components.WebView.WebView2.csproj" />
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView.WindowsForms" ProjectPath="$(RepoRoot)src\Components\WebView\Platforms\WindowsForms\src\Microsoft.AspNetCore.Components.WebView.WindowsForms.csproj" />
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView.Wpf" ProjectPath="$(RepoRoot)src\Components\WebView\Platforms\Wpf\src\Microsoft.AspNetCore.Components.WebView.Wpf.csproj" />
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView" ProjectPath="$(RepoRoot)src\Components\WebView\WebView\src\Microsoft.AspNetCore.Components.WebView.csproj" />
+    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.WebView" ProjectPath="$(RepoRoot)src\Components\WebView\src\Microsoft.AspNetCore.Components.WebView.csproj" />
```
Comment (SteveSandersonMS) [inline eng/ProjectReferences.props:153]: Would it be OK to leave this where it was so we can still have [other platforms](https://github.com/dotnet/aspnetcore/pull/33413)?

**Review by SteveSandersonMS — APPROVED** [review]
Other than my request to not move the `src` project, this all looks good!

I mean, really it actually looks awful because you're killing all our beautiful code, it's a total bloodbath - our babies! - but I'll get counselling and carry on.

**Comment by Eilon** [comment]
@dotnet/aspnet-build  - how do I fix this build failure?

```
Found changes in 3 API baseline files
##[error]Detected modification to baseline API files. PublicAPI.Shipped.txt files should only be updated after a major release. See /docs/APIBaselines.md for more information.
error : Detected modification to baseline API files. PublicAPI.Shipped.txt files should only be updated after a major release. See /docs/APIBaselines.md for more information.
##[error]Modified API baseline files:
error : Modified API baseline files:
##[error]src/Components/WebView/Platforms/WebView2/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/WebView2/src/PublicAPI.Shipped.txt
##[error]src/Components/WebView/Platforms/WindowsForms/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/WindowsForms/src/PublicAPI.Shipped.txt
##[error]src/Components/WebView/Platforms/Wpf/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/Wpf/src/PublicAPI.Shipped.txt

Summary:

   5 error(s)

error : Detected modification to baseline API files. PublicAPI.Shipped.txt files should only be updated after a major release. See /docs/APIBaselines.md for more information.
error : Modified API baseline files:
error : src/Components/WebView/Platforms/WebView2/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/WindowsForms/src/PublicAPI.Shipped.txt
error : src/Components/WebView/Platforms/Wpf/src/PublicAPI.Shipped.txt
```

I am deleting projects that never shipped RTM. They did ship as preview, but they are being deleted from here because they moved to another repo.

**Comment by wtgodbe** [comment]
If the only failure is that one in code-check, one of us can just use our admin privileges to merge this for you.

**Comment by Eilon** [comment]
@wtgodbe - thanks! I still have some work in this PR so I'll let you know when it gets to that point.

**Comment by dougbu** [comment]
@Eilon please also remove the now-useless block at https://github.com/dotnet/aspnetcore/blob/main/eng/Build.props#L46-L54

File: AspNetCore.sln [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.0.31324.12
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31320.298
```
Comment (TanayParikh) [inline AspNetCore.sln:4]: Is this intentional?

Comment (Eilon) [reply AspNetCore.sln:4]: I have unintentionally not yet installed VS17. But I will do that soon. I assume this header flip-flops all the time anyway?

File: eng/ProjectReferences.props (hunk 1 above)
Comment (Eilon) [reply eng/ProjectReferences.props]: Restored.

**Comment by Eilon** [comment]
@SteveSandersonMS the folder structure is restored, so this is a more clean removal now.

**Comment by Eilon** [comment]
@wtgodbe - can you super-admin merge this PR now? The code is done and it's signed off.

**Pull request merged**